src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -81,6 +108,20 @@ def validate_fields(cls, values):
             raise InternalError("Exactly one of `tensor_fields` or `non_tensor_fields` must be provided.")
 
         return values
+    
+    @validator('docs')
+    def validate_docs(cls, docs):
+        """
+        Validates that docs length does not exceed the maximum and is not empty.
+        More validation can be added in the future.
+        """
+        
+        docs = validate_add_docs_count(docs)
+        # TODO: insert future docs validation here
```

Comment:
nit: this comment is redundant because this information is already included in the docstring